packages/components/src/Enumeration/Items/Item/Item.component.js

```diff
@@ -61,12 +62,16 @@ function Item({ id, item }) {
 
 	return (
 		<li className={itemClasses(item.isSelected)} id={id}>
-			<Action
-				key={item.index}
+			<TooltipTrigger
 				label={item[key].join(',')}
-				onClick={event => onSelectItem(item, event)}
-				className={itemLabelClasses()}
-			/>
+				tooltipPlacement="top"
+			>
+				<Button
+					key={item.index}
+					onClick={event => onSelectItem(item, event)}
+					className={itemLabelClasses()}
+				>{item[key].join(',')}</Button>
+			</TooltipTrigger>
```

Comment:
I would prefer to keep Action component here. 
So please add a props to always render the tooltip: 

props.tooltip with PropTypes.boolean